Formula/jpeg-xl.rb

```diff
@@ -0,0 +1,34 @@
+class JpegXl < Formula
+  desc "New file format for still image compression"
+  homepage "https://jpeg.org/jpegxl/index.html"
+  url "https://gitlab.com/wg1/jpeg-xl.git",
+      tag:      "v0.2",
+      revision: "31c71b0f61123a40789b0b8f54feb70e5995420e"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "brotli"
```

Comment:
```suggestion
  depends_on "brotli"
  depends_on "gcc"
  depends_on "giflib"
  depends_on "ilmbase"
```
Is gcc needed at runtime? Or is it only a build dependency?